crates/aptos-crypto/src/bls12381/bls12381_pop.rs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+//! This module provides APIs for proofs-of-possesion (PoPs) used in BLS multi-signatures
+//! implemented on top of BLS12-381 elliptic curves (https://github.com/supranational/blst).
+//!
+//! PoPs were first introduced by Ristenpart and Yilek [^RY07].
+//!
+//! [^RY07]: The Power of Proofs-of-Possession: Securing Multiparty Signatures against Rogue-Key Attacks; by Ristenpart, Thomas and Yilek, Scott; in Advances in Cryptology - EUROCRYPT 2007; 2007
+
+use std::convert::TryFrom;
+use anyhow::{anyhow, Result};
+use blst::BLST_ERROR;
+use aptos_crypto_derive::{DeserializeKey, SerializeKey};
+use crate::bls12381::bls12381_keys::{PrivateKey, PublicKey};
+use crate::{CryptoMaterialError, Length, ValidCryptoMaterial, ValidCryptoMaterialStringExt};
+
+/// Domain separation tag (DST) for hashing a public key before computing its proof-of-possesion (PoP),
+/// which is also just a signature.
+const DST_BLS_POP_IN_G2: &[u8] = b"BLS_POP_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_";
+
+#[derive(Debug, Clone, Eq, PartialEq, SerializeKey, DeserializeKey)]
+/// A proof-of-possesion (PoP) of a BLS12381 private key.
+/// This is just a BLS signature on the corresponding public key.
+pub struct ProofOfPossession {
+    pub(crate) pop: blst::min_pk::Signature,
+}
+
+impl ProofOfPossession {
+    /// The length of a serialized ProofOfPossession struct
+    // NOTE: We have to hardcode this here because there is no library-defined constant
+    pub const LENGTH: usize = 96;
+
+    /// Serialize a ProofOfPossession.
+    pub fn to_bytes(&self) -> [u8; Self::LENGTH] {
+        self.pop.to_bytes()
+    }
+
+    /// Group-check the PoP: i.e., verifies the PoP is a valid group element.
```

Comment:
Replace the colon with a comma. As in "Group-checks the PoP, i.e., verifies the PoP..."